website/src/pages/tutorial/basic/08-graph-relations.mdx

```diff
@@ -376,12 +386,22 @@ type Link {
   id: ID!
   description: String!
   url: String!
-  comments: [Comment!]!
+  comments: [Comment!]
 }
 ```
 
+On the Prisma schema, we chose to make comments mandatory because of their best practices. So in
+case there are no comments, it will return an empty string.
```

Comment:
I an pretty sure it will not return an empty string.

```suggestion
On the Prisma API level, the method call for loading will always return an array. It does not distinguish between an empty array and an array with comments.
```